src/mailadm/db.py

```diff
@@ -8,6 +8,16 @@
 from .conn import Connection
 
 
+def read_connection():
+    mailadm_db = DB(get_db_path())
+    return mailadm_db.read_connection()
+
+
+def write_connection():
+    mailadm_db = DB(get_db_path())
+    return mailadm_db.write_transaction()
+
```

Comment:
These functions implicitely get the database path from global state (an environment variable).  Previously precise references were passed around.  I can see how you think it's easier to have global state but i consider it rather questionable (or even bad) practise.

Reply:
I reverted the changes I made.